web-common/src/features/dashboards/filters/measure-filters/measure-filter-entry.spec.ts

```diff
@@ -20,7 +20,7 @@ import { describe, expect, it } from "vitest";
 
 const TestCases: [
   title: string,
-  measureFilter: MeasureFilterEntry,
+  measureFilter: MeasureFilterEntry | undefined,
```

Comment:
Why are there test cases where `MeasureFilterEntry` is `undefined`? Is the point to test arbitrary filters passed through the URL? Or something else?

Reply:
Ya there is one case where we return undefined when it is not supported.